boards/rfduino/Makefile.features

```diff
@@ -0,0 +1,2 @@
+FEATURES_PROVIDED += cpp
+FEATURES_PROVIDED += periph_uart periph_gpio periph_random periph_rtt periph_cpuid
```

Comment:
https://github.com/isaacs/github/issues/168

yes this is my probem. i am new to this github thing. well

@thomaseichinger 

thomas, I ll clone a new riot soon, copy the files and changes from this pull - and create a new pull ? - is this ok?